### ISSUE TITLE
As a LOD user, I want the beng RDF schema to be focussed on content

### ISSUE THREAD

**Issue by mwigham** [beeldengeluid/beng-lod-server#53]
The DAAN model is focussed on carriers. Ideally we would like to shift this focus to be on the content. In particular because the model is complicated - you have multiple ways of linking content to a program:

- carrier -> technical logtrackitem ('normal' content)
- carrier -> scene description logtrackitem (clip in program)
- aggregation asset -> segment -> carrier ->technical logtrackitem ('normal' content split over carriers)
- aggregation asset -> segment -> carrier -> scene description logtrackitem (clip from a program split over carriers)

The model puts the technical implementation of where things are stored at the centre. In reality, most users won't care if content is split across carriers. These various different parts of the model also all come down to the same thing in essence - indicating a section of a carrier using start and duration - but this commonality is not clear from the model.

Enrichments are also stored as logtrackitems, which can be confusing as they are put at the same level as clips (which are content).

A potential solution is to model programs as having timelines. You could then have program content as one timeline, subtitling as another, term extraction as another etc. Each timeline consists of timeline elements with a start time and duration, which can each point to their carrier. A timeline can therefore contain items from the same carrier, or from different carriers, but will still be modelled in the same way. The elements can additionally contain an order indicator (needed for segments), descriptive metadata (needed for clips), or the content of enrichments (e.g. text for ASR).  

A new model would need to be able to handle all the different ways in which content can appear, without losing information. This could be particularly difficult with segments, where (I assume) all the logtrackitems on the carriers indicated by the segments must follow the ordering indicated by the segments. We would also need to consider what belongs together in a timeline - for example, should raw footage be in its own timeline?

**Comment by jblom** [comment]
@mwigham wow I can see now why your head was exploding when crunching the DAAN model...

hmm it might not be a crazy idea to store all "annotation like" logtracks as annotations. Also this would make the implementation of the front-end easier, since the video player with multiple timelines is there mostly for annotating segments of audio/video.

For now I kind of like the idea to:
1. model curated segments with possibly lots of metadata as one thing (in our own schema)
2. all "content annotations" (targetting a single media object) as W3C annotations

Also for the ES index we could say we just store the curated segments there and the content annotations in the annotation server.

Well anyway this is something for the whiteboard!

**Comment by wmelder** [comment]
We have been talking about this as early as the fist media suite versions... Back then we were not allowed to put effort in this ... But I agree that users don't care about carrier details and just want to see the part of the item they are interested in. But it is a bit complicated to model this... It is a more an epic than an issue anyway...
For inspiration: MPEG7 was a model that addressed this topic. But it seems that not everyone accepted the steep leaning curve... and the fact it was using XML extensively.

**Comment by wmelder** [comment]
reminds me that IIF AV is addressing this as well using canvases... https://iiif.io/api/cookbook/recipe/0065-opera-multiple-canvases/

**Comment by mwigham** [comment]
I agree that this is an epic. Interesting link from IIIF. Perhaps we can pick this up when we have a use case that requires this sort of information. So far the time-specific elements are not so important.
